<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>551</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="title">
     <string>Introduction</string>
    </property>
    <widget class="QComboBox" name="fctBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">alternate-background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Introduction:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>481</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="title">
     <string>Set Range</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>191</width>
       <height>371</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="userid_edit">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Segoe UI Historic&quot;;

</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="locationid_edit">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Segoe UI Historic&quot;;

</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="la_edit">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Segoe UI Historic&quot;;

</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lo_edit">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Segoe UI Historic&quot;;

</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QSplitter" name="splitter_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>161</width>
       <height>331</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QCheckBox" name="User_ID">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Arial&quot;;
selection-color: rgb(206, 206, 206);</string>
      </property>
      <property name="text">
       <string>User_ID</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Location_ID">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Arial&quot;;
selection-color: rgb(206, 206, 206);</string>
      </property>
      <property name="text">
       <string>Location_ID</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Latitude">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Arial&quot;;
selection-color: rgb(206, 206, 206);</string>
      </property>
      <property name="text">
       <string>Latitude</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Longitude">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Arial&quot;;
selection-color: rgb(206, 206, 206);</string>
      </property>
      <property name="text">
       <string>Longitude</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>810</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Go to Function</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>381</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/name.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/sign.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>601</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/see.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>40</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/mark.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>971</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/3.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_7" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>601</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/4.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>730</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/11.png);</string>
    </property>
   </widget>
   <zorder>widget_7</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>next</zorder>
   <zorder>widget_3</zorder>
   <zorder>widget</zorder>
   <zorder>widget_2</zorder>
   <zorder>widget_6</zorder>
   <zorder>widget_4</zorder>
   <zorder>widget_5</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_File">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
